AssetManagementSystem/Models/Asset.cs

```diff
@@ -1,21 +1,32 @@
 ﻿using System.ComponentModel.DataAnnotations;
+using System.ComponentModel.DataAnnotations.Schema;
 
 namespace AssetManagementSystem.Models
 {
     public class Asset
     {
         [Key]
+        //let's add this as well
```

Comment:
lets remove this comment aswell, since i put them to inform you

Reply:
Resolved